src/tree.rs

```diff
@@ -475,7 +491,7 @@ impl Data {
     }
 }
 
-
+/// `Tree` `Node` which stores the actual `Data` and metadata about itself and nearby `Node`s.
```

Comment:
`Tree` `Node` which is either an index or leaf, and has a pointer to the right sibling if there is one.